### PR TITLE
Vacation planner

### PR DESCRIPTION
**Pull request by vic85821**

### CODE REVIEW

File: VacationPlanner/Team1/BackpackingPlan.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+
+public class BackpackingPlan implements Plan{
+	private String day;
+	private String hotel;
+	private String tickets;
+	private String reservation;
+	private String events;
+	
+	public BackpackingPlan() {
+		day = new String("");
```
Comment (amjltc295) [inline VacationPlanner/Team1/BackpackingPlan.java]: Why is it 8 space ...

**Review by amjltc295 — COMMENTED** [review]
Please fix the possible tab; others look good to me

File: VacationPlanner/Team1/BackpackingPlan.java (hunk 1 above)
Comment (vic85821) [reply VacationPlanner/Team1/BackpackingPlan.java]: Typo :)

**Review by amjltc295 — APPROVED** [review]
LGTM

**Pull request merged**